src/NuGet.Server/Infrastructure/WebConfigSettingsProvider.cs

```diff
@@ -29,6 +29,13 @@ public bool GetBoolSetting(string key, bool defaultValue)
             return !bool.TryParse(settings[key], out value) ? defaultValue : value;
         }
 
+        public int GetIntSetting(string key, int defaultValue)
+        {
+            var settings = _getSettings();
+            int value;
+            return !int.TryParse(settings[key], out value) ? defaultValue : value;
```

Comment:
>out value) [](start = 48, length = 10)

Does this project support new C# language syntax? Can you `out var value` here?

Reply:
Should work.